projects/js-packages/publicize-components/src/components/connection-management/index.tsx

```diff
@@ -67,7 +67,6 @@ const ConnectionManagement = ( { className = null } ) => {
 
 	return (
 		<div className={ classNames( styles.wrapper, className ) }>
-			<h3>{ __( 'My Connections', 'jetpack' ) }</h3>
```

Comment:
I thought we want to keep this if they have connections, don't we?

Reply:
No, it was otherwise - p1716572179748519-slack-C02JJ910CNL.

@pablinos mentioned:

> it would look fine as just a button.